sos/report/plugins/openshift_ovn.py

```diff
@@ -20,18 +20,33 @@ class OpenshiftOVN(Plugin, RedHatPlugin):
     profiles = ('openshift',)
 
     def setup(self):
+        all_logs = self.get_option("all_logs")
+
         self.add_copy_spec([
             "/var/lib/ovn/etc/ovnnb_db.db",
             "/var/lib/ovn/etc/ovnsb_db.db",
-            "/var/lib/openvswitch/etc/keys",
-            "/var/log/openvswitch/libreswan.log",
-            "/var/log/openvswitch/ovs-monitor-ipsec.log"
-        ])
-        # Collect ovn interconnect specific files if exists.
+            "/var/lib/openvswitch/etc/keys"
+        ], sizelimit=300)
+
+        # Collect ovn interconnect specific db files if exists.
         self.add_copy_spec([
             "/var/lib/ovn-ic/etc/ovnnb_db.db",
             "/var/lib/ovn-ic/etc/ovnsb_db.db"
-        ])
+        ], sizelimit=300)
+
+        # Collect ipsec logs for openvswitch and additionally libovsdb
+        # logs in case of ovn interconnect setup.
+        if not all_logs:
+            self.add_copy_spec([
+                "/var/log/openvswitch/libreswan.log",
+                "/var/log/openvswitch/ovs-monitor-ipsec.log"
```

Comment:
why was it moved here? I would say we need these logs even when all_logs is set?

Reply:
`/var/log/openvswitch` is already being captured by the `openvswitch` plugin. Would that not be activated?

Reply:
I have seen several such feedbacks that a PR wants to collect a copypath in *another* plugin; maybe does it make sense to have a CI test for it? Though it can be tricky to have it working properly (e.g. due to multiline a_s_c, asterisks in paths, paths with variables etc)?

Reply:
I think it should be collected for openvswitch plugin, but I don't think that happens for some reason, may be useful to check/debug that

Reply:
ok @npinaeva , I will check.

Reply:
@npinaeva This works as expected as per openvswitch plugin logic [here](https://github.com/sosreport/sos/blob/main/sos/report/plugins/openvswitch.py#L66-L71) on an ovn cluster, in fact we can remove collecting ipsec logs here as openvswitch plugin already taking care of collecting `*.log` when` --all-logs` not specified.

Reply:
nice, thanks for checking! then we can just remove these files

Reply:
removed collecting ipsec logs from `/var/log/openvswitch` directory.